Vagrantfile

```diff
@@ -13,7 +13,10 @@ Vagrant.configure(2) do |config|
   # Forward ipython notebook's port to the host
   config.vm.network "forwarded_port", guest: 8888, host: 8888
 
-  config.vm.provision "shell", inline: <<-SHELL
+  # Get VirtualBox version installed on the host
+  VBOX_VERSION = `VirtualBox -help | head -1 | cut -d' ' -f5`
+
+  config.vm.provision "shell", args: "#{VBOX_VERSION}", inline: <<-SHELL
     sudo apt-get update
```

Comment:
Should we do `set -e` here to have the script fail when any command in this fails?  I tried running this and it failed to download the file (404).  Afterwards it tried to mount the iso that it hadn't downloaded, and launch VBoxLinuxAdditions.run and,...

With `-e` at least when a command fails we stop the execution.  This doesn't belong to this commit, but I think it merits a new commit.  Note that the output for failures is less than perfect because vagrant sends all the output again for whatever reason, but still, you get the script to fail as soon as there is a filure.